docs/specs/clients/notify/rpc-methods.md

```diff
@@ -52,18 +149,101 @@ Used to subscribe notify subscription to a peer through subscribe topic. Respons
 | Tag     | 4001     |
 ```
 
-### wc_notifyMessage
+Topic: notify topic
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant U AS Wallet
+  participant W AS SDK
+  participant I AS Identity Server
+  participant N AS Notify
+  participant D as Notify DB
+  participant A as dApp
+
+  activate U
+  U->>+W: User opens app
+  Note over W: Prepare request
+  Note over W: notify_subscribe = signJwt(params, identityKey)
+  W->>+N: notify_subscribe + publicKeyY ON reqTopic
+  Note over N: Generate P
+  N->>+D: Store P @ domain,account
+  Note over N,D: Error if sub already exists.
+  D-->>-N: ACK
+  Note over N: response = signJwt(P, identityKey)
+  N->>-W: response ON resTopic
+  W->>+R: subscribe(sha256(P))
+  R-->>-W: ACK
+  W->>-U: UI updates
+  deactivate U
+```
 
-Used to publish a notification message to a peer through notify topic. Response is expected on the same topic.
+## Detailed request & response pattern
+
+This is used by `wc_notifyWatchSubscribe` and `wc_notifySubscribe`
```

Comment:
```suggestion
This is used by `wc_notifyWatchSubscriptions` and `wc_notifySubscribe`
```

Reply:
Fixed in 68042ade3f27b44cf1a7e5742121168ab127bc47